configuration_go/manifests/compactor-tenant-a/compactor-statefulSet.yaml

```diff
@@ -0,0 +1,176 @@
+              key: name
+              name: aws_access_key_id
+        - name: AWS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              key: name
+              name: aws_secret_access_key
+        image: quay.io/thanos/thanos:v0.32
+        imagePullPolicy: IfNotPresent
+        livenessProbe:
+          failureThreshold: 4
+          httpGet:
+            path: /-/healthy
+            port: 10902
+          periodSeconds: 30
+        name: thanos
+        ports:
+        - containerPort: 10902
+          name: http
+          protocol: TCP
+        readinessProbe:
+          failureThreshold: 20
+          httpGet:
+            path: /-/healthy
+            port: 10902
+          periodSeconds: 5
+        resources:
+          limits:
+            cpu: "3"
+            memory: 3000Mi
+          requests:
+            cpu: "2"
+            memory: 2000Mi
+        terminationMessagePolicy: FallbackToLogsOnError
+      - args:
+        - -https-address=:8443
+        - -upstream=http://localhost:10902
+        - -openshift-service-account=observatorium-thanos-compact
+        - -tls-cert=/etc/tls/private/tls.crt
+        - -tls-key=/etc/tls/private/tls.key
+        - -client-secret-file=/var/run/secrets/kubernetes.io/serviceaccount/token
+        - -cookie-secret-file=/etc/proxy/secrets/session_secret
+        - -openshift-ca=/etc/pki/tls/cert.pem
+        - -openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        image: quay.io/oauth-proxy:4.13.0
+        imagePullPolicy: IfNotPresent
+        name: oauth-proxy
+        ports:
+        - containerPort: 8443
+          name: https
+        resources:
+          limits:
+            cpu: 50m
+            memory: 200Mi
+          requests:
+            cpu: 25m
+            memory: 200Mi
+        terminationMessagePolicy: FallbackToLogsOnError
+        volumeMounts:
+        - mountPath: /etc/tls/private
+          name: compact-tls
+        - mountPath: /etc/proxy/secrets
+          name: compact-proxy
+      nodeSelector:
+        kubernetes.io/os: linux
+      securityContext:
+        fsGroup: 65534
+        runAsUser: 65534
+      serviceAccountName: compactor-tenant-a
+      terminationGracePeriodSeconds: 120
+      volumes:
+      - name: compact-tls
+        secret:
+          secretName: compact-tls
+      - name: compact-proxy
+        secret:
+          secretName: compact-proxy
+  updateStrategy: {}
+  volumeClaimTemplates:
+  - metadata:
+      creationTimestamp: null
+      labels:
+        app.kubernetes.io/component: database-compactor
+        app.kubernetes.io/instance: observatorium
+        app.kubernetes.io/name: thanos-compact
+        app.kubernetes.io/part-of: observatorium
+        app.kubernetes.io/version: v0.32
+      name: data
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 50Gi
+      storageClassName: gp2-csi
+    status: {}
+status:
```

Comment:
is it possible to strip status from the output? This cant be supplied by the client

Reply:
Yep, totally.  Just an observation and comment as opposed to being a problem.